packages/dev/core/src/ShadersWGSL/edgeDetection.fragment.fx

```diff
@@ -0,0 +1,69 @@
+precision highp float;

```

Comment:
This file should also be moved

Reply:
I should've checked again
Sorry. It's my mistake😅

Reply:
But it seems that the pipeline times out after changing the path.
What am I have to do?